stacks-signer/src/v0/signer.rs

```diff
@@ -865,6 +1002,14 @@ impl Signer {
             }
         }
         self.broadcast_signed_block(stacks_client, block_info.block, &addrs_to_sigs);
+        if self
```

Comment:
Can you add a comment here explaining why this makes sense?

Reply:
Done!